DataStructures/Graphs/Dijkstras_algorithm/MinimumDistance.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class MinimumDistance {
+
+	public static void main(String[] args) {
+		int n, m;
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter number of vertices: ");
+		n = scanner.nextInt();
```

Comment:
Provide some test cases for the implementation.

Reply:
I have added a text file TestCase.txt that contains a set of 3 test cases. Each test case is labeled as Test case #<test_case_number>. Do let me know if you need more test cases.